src/transformers/modeling_tf_transfo_xl.py

```diff
@@ -734,7 +734,7 @@ def call(self, inputs, **kwargs):
         return outputs
 
 
-class TFTransfoXLLMHead(tf.keras.layers.Layer):
+class TFTransfoXLWithLMHeadModel(tf.keras.layers.Layer):
```

Comment:
This is a breaking change, no? Maybe we want to add an alias for `TFTransfoXLLMHead` for backward compatibility @LysandreJik

Reply:
It is my bad, I just done a commit to rename it.